src/analysis/convex/basic.lean

```diff
@@ -29,207 +28,388 @@ We also provide various equivalent versions of the definitions above, prove that
 are convex, and prove Jensen's inequality.
 
 Note: To define convexity for functions `f : E → β`, we need `β` to be an ordered vector space,
-defined using the instance `ordered_smul ℝ β`.
+defined using the instance `ordered_smul 𝕜 β`.
 
 ## Notations
 
-We use the following local notations:
-
-* `I = Icc (0:ℝ) 1`;
-* `[x, y] = segment x y`.
+We use the following notations:
+* `[x -[𝕜] y] = segment 𝕜 x y`
+* `I = Icc (0 : ℝ) 1`
 
-They are defined using `local notation`, so they are not available outside of this file.
+The second one is defined using `local notation`, so it is not available outside of this file.
```

Comment:
While you're at it, could you upgrade any local notation to a locale, so it is available outside this file with an appropriate `open_locale ...`?